api/src/models/plan.js

```diff
@@ -75,6 +80,10 @@ const planSchema = mongoose.Schema({
         type: Boolean,
         required: true,
     },
+    notes: {
+        type: [String],
+        require: false
```

Comment:
shouldn't it be `required`, not `require`

Reply:
oh yikes forgot to push that change